src/SharedPipelineComponents/IlrSubmissionEvents/DeploySubmission/sql/migrationscripts/~2_Submissions.Deds.AddColumns.sql

```diff
@@ -14,4 +14,37 @@ IF NOT EXISTS(SELECT [column_id] FROM sys.columns WHERE [name] = 'EPAOrgId' AND
     ALTER TABLE [Submissions].[SubmissionEvents]
 	ADD [EPAOrgId] varchar(7) NULL
   END
+GO
+
```

Comment:
You'll need to move this into a new migration script as number 2 will already have ran.